### DISCUSSION TITLE
Winver-generic icon for "Apps" folder

### DISCUSSION THREAD

**Discussion by bonzibudd** [Open-Shell/Open-Shell-Menu#1260] [General]
Another small project I'd like to do; The current icon has the Windows 7 folder built in, which looks very out of place on newer versions. I think I'd like to change this to an icon similar to the current design, but without the folder behind it. It would be nice to have a folder icon that changes depending on the version, but I don't know how to implement that.

**Comment by ge0rdi** [comment]
You can prepare such icon(s) and create feature request to add them.
I think it should be quite easy to add.

**Reply by bonzibudd** [reply]
Do you mean that you would be able to assign certain icons to different OS versions? If so, I'll go ahead and start working on version specific icons.

**Reply by ge0rdi** [reply]
Yup.
It would have to be changed on more places (actual menu, custom commands in settings, ...) but I guess it should be possible.
